weave-js/wb_schema.gql

```diff
@@ -0,0 +1,605 @@
+# This file is a hand-selected subset of the W&B GQL spec that is used by weave-js.
+# These type definitions are used to generate TypeScript types for GQL operations.
+# If there's a new operation you want to support in weave-js, you must add the
+# corresponding type definitions here.
```

Comment:
As mentioned [above](https://github.com/wandb/weave/pull/674#discussion_r1342969555), this file just copies an existing file, except I updated this comment text for the weave-js context

Reply:
hm... seeing this now - i wonder if we can just reuse the same file as we have for testing? .. cc @dannygoldstein

Reply:
If I try to point the codegen config to a file outside of the weave-js dir, this error occurs while building the integration test image: https://github.com/wandb/weave/actions/runs/6384193261/job/17326341162#step:6:286

I am in favor of using a shared schema file across weave, but need a little help resolving the build issue

Reply:
Update -- we're now using 1 schema file across FE & BE!